packages/inertia-react/index.d.ts

```diff
@@ -58,6 +64,8 @@ export const InertiaLink: InertiaLink
 
 export const Link: InertiaLink
 
+export const InertiaHead: InertiaHead
```

Comment:
I think you are missing a `export const Head: InertiaHead` since the InertiaHead component is also exported as Head

Reply:
You are totally right. I added it now